src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: New Relic discussions and commenting integration with Slack
+tags:
+  - New Relic 
+  - Use New Relic
+  - Get started
+metaDescription: How to use our new collaboration features with Slack
+---
+import post_on_slack from 'images/post_on_slack.png'
+
+import slack_app_management from 'images/slack_app_management.png'
+
+import slack_auth from 'images/slack_auth.png'
+
+import slack_synced from 'images/slack_synced.png'
+
+import slack_channel from 'images/slack_channel.png'
+
+While not everyone you work with is a New Relic user, most of your teammates are probably on Slack. Now you can discuss New Relic data with everyone and keep all insights synced between Slack and New Relic discussions.
+
+Unlike many Slack integrations that just share an update to Slack in one-direction, conversation threads on New Relic are synced in both directions with threads on Slack. When you share a question, observation, or request for assistance from New Relic to Slack, your teammates on Slack—whether they are New Relic users themselves or not—can reply to the post on Slack and these replies will be brought back into the thread on New Relic. Likewise, replies added to the thread on New Relic will be added to the thread on Slack. This back and forth sync lets you include more people in the conversation, capture the discussion along with the context in New Relic, use data to inform decisions, and build institutional knowledge. All at the same time.
+
+## Installation [#install]
+
+The New Relic discussions Slack app is not yet in the Slack App Directory. Depending on your Slack workspace's **App Management Settings**, you may need the assistance of your Slack administrator for installation. 
+
+If set, your Slack workspace admin may need to “Turn off “Only allow apps from the Slack App Directory,”, and likewise turn on “Allow members to request approval for apps.”. Then you can proceed with the authentication instructions below. Following those instructions will generate a request for an admin to approve the application after which, you can install the Slack app, and any settings temporarily changed can be restored.
```

Comment:
I think you can move the comma that's after "after which" to after "application"